lib/common/json-schema-validator.js

```diff
@@ -90,25 +95,105 @@ function jsonSchemaValidatorFactory(
     };
 
     /**
-     * Get compiled schema from the instance by `key` or `ref`.
-     * @param  {String} key `key` that was passed to `addSchema` or
-     *                  full schema reference (`schema.id` or resolved id).
+     * Add schema to the instance.
+     * @param {Object} schema
+     * @param {String} name Optional schema name when schema has an id
```

Comment:
To describe the optional argument, you can use
```
@param {String} [name] - Optional schema name
```
Reference: http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values